app/_includes/plugins/logging/log-format.md

```diff
@@ -2,6 +2,13 @@
 
 {% new_in 3.7 %} Every request is logged separately in a JSON object, separated by a new line `\n`, with the following format:
```

Comment:
Reviewers: with the log example codeblock being collapsed and the object description table more prominent, with new_in labels - do we need this 3.7 badge on this line? I would vote against it.